CHANGELOG.md

```diff
@@ -1,8 +1,34 @@
 ## Next release
 
+## 0.39.0 / 2020-05-06
+
+* [CHANGE] Introduce release schedule (#3135)
+* [CHANGE] Remove Custom Resource Definition management and move CRD definitions to bundle.yaml (#3155, #3171)
+* [CHANGE] Remove apiextensions.k8s.io/v1beta1 CRD and require kubernetes v1.16 or newer (#3175, #3187)
+* [FEATURE] Add support prometheus query log file (#3116)
+* [FEATURE] Add support for watching specified rules directory by config-relader (#3128)
+* [FEATURE] Add TLS support for operator web server (#3134, #3157)
+* [FEATURE] Allow to set address for operator http endpoint (#3098)
+* [FEATURE] Allow setting the alertmanagers cluster.advertiseAddress (#3160)
+* [FEATURE] Add support of matching only PodMonitors (#3173)
+* [FEATURE] Build operator images for ARM and ARM64 platforms (#3177)
+* [ENHANCEMENT] Allow setting log level and format for thanos sidecar (#3112)
+* [ENHANCEMENT] Support naming of remote write queues (#3144)
+* [ENHANCEMENT] Allow disabling mount subPath for volumes (#3134)
+* [ENHANCEMENT] Update k8s libraries to v1.18 (#3154)
+* [ENHANCEMENT] Build tooling improvements and updates (#3108, #3109, #3132, #3141 ,#3174)
```

Comment:
ditto

Reply:
I was also wondering if this is necessary, but we did the same in v0.36.0 :shrug:

Reply:
Sure, but that doesn't mean we can't improve now :) CHANGELOG should contain user facing changes, tooling is internal and end users don't care much.